pom.xml

```diff
@@ -1331,6 +1336,28 @@
             </exclusions>
           </dependency>
           <dependency>
+          	<groupId>org.apache.avro</groupId>
+          	<artifactId>avro</artifactId>
+          	<version>1.7.4</version>
+          	<exclusions>
+          		<exclusion>
+          			<groupId>org.xerial.snappy</groupId>
+          			<artifactId>snappy-java</artifactId>
+          		</exclusion>
+          	</exclusions>
+          </dependency>
+          <dependency>
+          	<groupId>org.apache.parquet</groupId>
+          	<artifactId>parquet-hadoop</artifactId>
+          	<version>1.8.1-drill-r0</version>
```

Comment:
Can we use ${parquet.version} here, which is defined in the beginning of root/pom.xml?  It will save effort, in case Drill changes version number for parquet library.